draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -379,23 +380,41 @@ Algorithms in this document make use of utility functions described below.
 - round(x): floor(ceil(2\*x)/2), i.e. rounding to the nearest
```

Comment:
This definition is quite complicated. The number of bits of a positive integer is floor(log2(x))+1.